lib/Net/SSLeay.pod

```diff
@@ -4625,6 +4625,32 @@ When callback is undef, an existing callback will be disabled.
 
 Check openssl doc L<http://www.openssl.org/docs/ssl/SSL_CTX_set_info_callback.html|http://www.openssl.org/docs/ssl/SSL_CTX_set_info_callback.html>
 
+=item * set_msg_callback
+
+Sets the callback function, that can be used to obtain protocol messages information for $ssl during connection setup and use.
+When callback is undef, the callback setting currently valid for ctx is used.
+
+ Net::SSLeay::set_msg_callback($ssl, $cb, [$data]);
+ # $ssl - value corresponding to openssl's SSL structure
+ # $cb - sub { my ($ssl,$write_p,$version,$content_type,$buffer,$len,$data) = @_; ... }
+ #
+ # returns: no return value
+
+Check openssl doc L<http://www.openssl.org/docs/ssl/SSL_set_msg_callback.html|http://www.openssl.org/docs/ssl/SSL_set_msg_callback.html>
+
+=item * CTX_set_msg_callback
+
+Sets the callback function on ctx, that can be used to obtain protocol messages information for ssl connection setup and use.
+When callback is undef, the existing callback will be disabled.
+
+ Net::SSLeay::CTX_set_msg_callback($ssl, $cb, [$data]);
+ # $ssl - value corresponding to openssl's SSL structure
+ # $cb - sub { my ($ssl,$write_p,$version,$content_type,$buffer,$len,$data) = @_; ... }
+ #
+ # returns: no return value
+
+Check openssl doc L<http://www.openssl.org/docs/ssl/CTX_SSL_set_msg_callback.html|http://www.openssl.org/docs/ssl/CTX_SSL_set_msg_callback.html>
```

Comment:
The prefix is a bit incorrect (SSL_CTX, not CTX_SSL) https://www.openssl.org/docs/manmaster/man3/SSL_CTX_set_msg_callback.html would be correct here.

Reply:
Heh, thx! The number of times I typed that in wrongly. I've pushed those changes now.